### PR TITLE
adding first WebNLG prompts.

### PR DESCRIPTION
**Pull request by jordiclive**
WebNLG prompts.

### CODE REVIEW

File: promptsource/templates/enriched_web_nlg/en/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    answer_choices: null
+    id: 715a885b-1022-43b0-bcfe-20fa432314da
+    jinja: 'Take the following graph comprising triple sets, where each element of
+      a triple is separated  by "|" and each triple set by ",": {{modified_triple_sets.mtriple_set[0]
+      | join(", ")}}. Make a verbalization of the triple set into plain text. |||
+      {{lex.text[0]}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
```
Comment (VictorSanh) [inline promptsource/templates/enriched_web_nlg/en/templates.yaml:39]:
```suggestion
      - BLEU
      - ROUGE
```

File: promptsource/templates/enriched_web_nlg/en/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      original_task: true
+    name: verbalize by commas
+    reference: 'instruction: verbalization'
+  715a885b-1022-43b0-bcfe-20fa432314da: !Template
+    answer_choices: null
+    id: 715a885b-1022-43b0-bcfe-20fa432314da
+    jinja: 'Take the following graph comprising triple sets, where each element of
+      a triple is separated  by "|" and each triple set by ",": {{modified_triple_sets.mtriple_set[0]
+      | join(", ")}}. Make a verbalization of the triple set into plain text. |||
+      {{lex.text[0]}}'
```
Comment (VictorSanh) [inline promptsource/templates/enriched_web_nlg/en/templates.yaml:35]:
```suggestion
      {{lex.text | choice}}'
```

File: promptsource/templates/enriched_web_nlg/en/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+dataset: enriched_web_nlg
+subset: en
+templates:
+  3860d7fb-0b50-4275-a7ab-782ae86756e5: !Template
+    answer_choices: null
+    id: 3860d7fb-0b50-4275-a7ab-782ae86756e5
+    jinja: 'Take the following triple set as part of a Data-to-Text task: {{modified_triple_sets.mtriple_set[0]
+      | join(", ")}}. Make a lexicalization of the triple set into plain text. |||
+      {{lex.text[0]}}'
```
Comment (VictorSanh) [inline promptsource/templates/enriched_web_nlg/en/templates.yaml:9]:
```suggestion
      {{lex.text | choice}}'
```

File: promptsource/templates/enriched_web_nlg/en/templates.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      metrics:
+      - BLEU
+      original_task: true
+    name: non-explicit lexicalization
+    reference: 'Triple set: lexicalization'
+  5c203187-70e4-4913-86af-8b00b5ca9e16: !Template
+    answer_choices: null
+    id: 5c203187-70e4-4913-86af-8b00b5ca9e16
+    jinja: 'Verbalize the following triples separated by a comma: {{modified_triple_sets.mtriple_set[0]
+      | join(", ")}} ||| {{lex.text[0]}}'
```
Comment (VictorSanh) [inline promptsource/templates/enriched_web_nlg/en/templates.yaml:21]:
```suggestion
      | join(", ")}} ||| {{lex.text | choice}}'
```

File: promptsource/templates/enriched_web_nlg/en/templates.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      original_task: true
+    name: explicit instruction
+    reference: 'Triple set, Verbalize:'
+  e80f68dd-ebd0-4cbc-960d-bb28aff2d2d4: !Template
+    answer_choices: null
+    id: e80f68dd-ebd0-4cbc-960d-bb28aff2d2d4
+    jinja: 'Take the following graph comprising triple sets, where each element of
+      a triple is separated  by "|" and each triple set by ",": {{modified_triple_sets.mtriple_set[0]
+      | join(", ")}}. Make a verbalization of the triple set into plain text. |||
+      {{lex.text[0]}}'
```
Comment (VictorSanh) [inline promptsource/templates/enriched_web_nlg/en/templates.yaml:49]:
```suggestion
      {{lex.text  | choice}}'
```

File: promptsource/templates/enriched_web_nlg/en/templates.yaml [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    reference: 'Triple set: lexicalization'
+  5c203187-70e4-4913-86af-8b00b5ca9e16: !Template
+    answer_choices: null
+    id: 5c203187-70e4-4913-86af-8b00b5ca9e16
+    jinja: 'Verbalize the following triples separated by a comma: {{modified_triple_sets.mtriple_set[0]
+      | join(", ")}} ||| {{lex.text[0]}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
```
Comment (VictorSanh) [inline promptsource/templates/enriched_web_nlg/en/templates.yaml:25]:
```suggestion
      - BLEU
      - ROUGE
```

File: promptsource/templates/enriched_web_nlg/en/templates.yaml [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    answer_choices: null
+    id: e80f68dd-ebd0-4cbc-960d-bb28aff2d2d4
+    jinja: 'Take the following graph comprising triple sets, where each element of
+      a triple is separated  by "|" and each triple set by ",": {{modified_triple_sets.mtriple_set[0]
+      | join(", ")}}. Make a verbalization of the triple set into plain text. |||
+      {{lex.text[0]}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
```
Comment (VictorSanh) [inline promptsource/templates/enriched_web_nlg/en/templates.yaml:53]:
```suggestion
      - BLEU
      - ROUGE
```

File: promptsource/templates/enriched_web_nlg/en/templates.yaml [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+      original_task: true
+    name: explicit instruction verbalization
```
Comment (VictorSanh) [inline promptsource/templates/enriched_web_nlg/en/templates.yaml:55]: What is the difference between this template and `explicit instruction`?

File: promptsource/templates/enriched_web_nlg/en/templates.yaml [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+  3860d7fb-0b50-4275-a7ab-782ae86756e5: !Template
+    answer_choices: null
+    id: 3860d7fb-0b50-4275-a7ab-782ae86756e5
+    jinja: 'Take the following triple set as part of a Data-to-Text task: {{modified_triple_sets.mtriple_set[0]
+      | join(", ")}}. Make a lexicalization of the triple set into plain text. |||
+      {{lex.text[0]}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
```
Comment (VictorSanh) [inline promptsource/templates/enriched_web_nlg/en/templates.yaml:13]:
```suggestion
      - BLEU
      - ROUGE
```

**Review by VictorSanh — COMMENTED** [review]
Thanks for our contribution @jordiclive !
It looks good to me, I made some comments and edits, please have a look :)
The tests are fine locally (datasets upgrade problem see #732 )

File: promptsource/templates/enriched_web_nlg/en/templates.yaml (hunk 9 above)
Comment (jordiclive) [reply promptsource/templates/enriched_web_nlg/en/templates.yaml:13]: ROUGE is not usually reported for Data-to-Text, usually METEOR, TER, but do see it in the E2E paper 👍 so think a good change.

Comment (jordiclive) [reply promptsource/templates/enriched_web_nlg/en/templates.yaml:13]: Would be good to have multi-reference BLEU to use the references.

**Comment by jordiclive** [comment]
Thank you! I have updated.

**Comment by VictorSanh** [comment]
thank you @jordiclive! merging now :)

**Pull request merged**